bankingapp/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+
+    environment:
+      - POSTGRES_PASSWORD="${DB_PASS}"
+
+    volumes:
+      - data:/var/lib/postgresql/data
+
+
+  bankingapp:
+    build: ./.
+
+    depends_on:
+      - db
+
+    volumes:
+      - ./.:/home/bankingapp/
+    ports:
+      - "8000:8000"
```

Comment:
You should mention the location of the .env file here. Look into how to use .env files using docker-compose